net.waterfox.waterfox.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: net.waterfox.waterfox
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: waterfox
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--filesystem=xdg-download:rw"
+  - "--filesystem=~/.waterfox:create"
```

Comment:
You should use --persist instead.

Reply:
I've switched my test build to use --persist, and it works just fine, but where does it store the data?

Reply:
I found the data location.